src/Cards/Cards.js

```diff
@@ -30,14 +32,37 @@ export default class Cards extends Component {
     });
   }
 
+  sortMovies = (value) => {
+    // console.log(value)
+    if (!value || value === '--none--') {
+      return false
+    } else if (value === 'average_rating_descending'){
+      const sortedMovies = this.state.movies.sort((a, b) => a.average_rating - b.average_rating)
+      this.setState({ movies: sortedMovies })
+    } else if (value === 'average_rating_ascending'){
```

Comment:
I think because there are only 3 options here `if (!value || value === '--none--')`, `else if (value === 'average_rating_descending')`, and `else if (value === 'average_rating_ascending')`, one of them can be turned into just an else statement. Other than that, this function looks solid!